packages/gatsby/src/datastore/types.ts

```diff
@@ -1,35 +1,52 @@
 import { Database } from "lmdb-store"
 import { IGatsbyNode } from "../redux/types"
+import { GatsbyGraphQLType } from "../../index"
+import { IInputQuery } from "./common/query"
+import { IGraphQLRunnerStats } from "../query/types"
+import { GatsbyIterable } from "./common/iterable"
 
 export type NodeId = string
 export type NodeType = string
 
 export interface ILmdbDatabases {
   nodes: Database<IGatsbyNode, NodeId>
   nodesByType: Database<NodeId, NodeType>
+  indexes: Database<NodeId, Array<any>>
+  metadata: Database<any, string>
```

Comment:
this change seems like one from future PRs (one that actually adds indexes to lmdb)? No need to revert, just checking (can't find use of those just yet in here)

Reply:
Yeah, this is a stray change from the other PR. I'll remove it from this PR